cmake/modules/FindCordova.cmake

```diff
@@ -0,0 +1,8 @@
+find_program(UPM_CORDOVA_BINDING NAMES upm-cordova-binding
+)
+
+if(UPM_CORDOVA_BINDING)
+    message(STATUS "The npm package to build cordova bindings found at ${UPM_CORDOVA_BINDING}")
```

Comment:
Maybe we should add a version declaration here as well, as Yuidoc do https://github.com/intel-iot-devkit/upm/blob/master/cmake/modules/FindYuidoc.cmake#L16 and check for the version as well